website/i18n/fr/docusaurus-plugin-content-docs/current/concepts/combining_providers.mdx

```diff
@@ -57,31 +57,33 @@ class TodoList extends StateNotifier<List<Todo>> {
 
 final todoListProvider = StateNotifierProvider((ref) => TodoList());
 ```
-Un cas d'utilisation commun serait de faire en sorte que l'interface utilisateur (UI) filtre la liste des tâches (todos) 
+
+Un cas d'utilisation commun serait de faire en sorte que l'interface utilisateur (UI) filtre la liste des tâches (todos)
 pour n'afficher seulement les tâches terminées/non terminées.
 
 Une façon simple de mettre en place un tel scénario serait de :
 
 - créer [StateProvider], qui présente la méthode de filtrage actuellement sélectionnée :
 
- ```dart
-  enum Filter {
-    none,
-    completed,
-    uncompleted,
-  }
+```dart
```

Comment:
The indentation should stay

Reply:
Ok thank you and sorry again! Going to fix it now God Willing!